ElunaLoader.h

```diff
@@ -10,11 +10,23 @@
 
 #include "ElunaUtility.h"
 
+#ifdef TRINITY
+#include <filesystem>
```

Comment:
Nothing in this diff uses filesystem header (and the preferred filesystem implementation in TrinityCore is boost/filesystem)

Reply:
You're right, it's a remnant from the original patch, and is covered by the boost fs header in the cpp file. Will remove the include now.